connection.go

```diff
@@ -207,25 +210,27 @@ type Opts struct {
 // Address could be specified in following ways:
 //
 // TCP connections:
-// - tcp://192.168.1.1:3013
-// - tcp://my.host:3013
-// - tcp:192.168.1.1:3013
-// - tcp:my.host:3013
-// - 192.168.1.1:3013
-// - my.host:3013
+//  - tcp://192.168.1.1:3013
+//  - tcp://my.host:3013
+//  - tcp:192.168.1.1:3013
+//  - tcp:my.host:3013
+//  - 192.168.1.1:3013
+//  - my.host:3013
+//
 // Unix socket:
-// - unix:///abs/path/tnt.sock
-// - unix:path/tnt.sock
-// - /abs/path/tnt.sock  - first '/' indicates unix socket
-// - ./rel/path/tnt.sock - first '.' indicates unix socket
-// - unix/:path/tnt.sock  - 'unix/' acts as a "host" and "/path..." as a port
+//  - unix:///abs/path/tnt.sock
+//  - unix:path/tnt.sock
+//  - /abs/path/tnt.sock  - first '/' indicates unix socket
+//  - ./rel/path/tnt.sock - first '.' indicates unix socket
+//  - unix/:path/tnt.sock - 'unix/' acts as a "host" and "/path..." as a port
 //
 // Note:
 //
 // - If opts.Reconnect is zero (default), then connection either already connected
 // or error is returned.
 //
-// - If opts.Reconnect is non-zero, then error will be returned only if authorization// fails. But if Tarantool is not reachable, then it will attempt to reconnect later
+// - If opts.Reconnect is non-zero, then error will be returned only if authorization
```

Comment:
Lists like these seem weird in godoc

![image](https://user-images.githubusercontent.com/20455996/161788214-bab7148b-44a9-4a8d-b3ca-cd3da318ed67.png)